### PR TITLE
chore: replace tslint with eslint

### PR DESCRIPTION
**Pull request by Uzlopak**
self explaining

### CODE REVIEW

File: src/compiler/header-builder.ts [hunk 1]
```diff
@@ -67,13 +68,13 @@ export class HeaderBuilder {
...
-      'const char* p, const char* endp);\n';
+    'const char* p, const char* endp);\n';
 
     res += '\n';
     res += '#ifdef __cplusplus\n';
-    res += '}  /* extern "C" *\/\n';
+    res += '}  /* extern "C" */\n';
     res += '#endif\n';
-    res += `#endif  /* ${DEFINE} *\/\n`;
+    res += `#endif  /* ${DEFINE} */\n`;
```
Comment (anonrig) [inline src/compiler/header-builder.ts:77]: what's the reason for the change of this line?

Comment (Uzlopak) [reply src/compiler/header-builder.ts:77]: Its an unnecessary escape character. eslint complains about it. checked it and it is true.

**Review by anonrig — APPROVED** [review]
